recipes/hippomocks/all/conanfile.py

```diff
@@ -0,0 +1,39 @@
+from conans import ConanFile, ConanFile, tools
+import os, glob
+
+class HippomocksConan(ConanFile):
+    name = 'hippomocks'
+    version = '10960d0' # This library is not versioned
```

Comment:
Please, read and follow: https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#what-version-should-packages-use-for-libraries-without-official-releases